src/context/alert.tsx

```diff
@@ -92,6 +93,7 @@ const messageToTranslationKeyMappings: Record<string, string> = {
   [ERROR_MESSAGE.MISSING_VOUCHER_INPUT]: "incompleteEntryVoucherCode",
   [ERROR_MESSAGE.INVALID_POD_INPUT]: "wrongFormatNotValidDeviceCode",
   [ERROR_MESSAGE.MISSING_POD_INPUT]: "incompleteEntryScanDeviceCode",
+  [ERROR_MESSAGE.INVALID_IDENTIFIER]: "invalidDeviceCode",
```

Comment:
Might be better to change this to:

```suggestion
  [ERROR_MESSAGE.INVALID_POD_IDENTIFIER]: "invalidDeviceCode",
```

since we know this is specific to pod! 

Let me know what you think!

Reply:
okk